lib/geoengineer/resources/aws/sqs/aws_sqs_queue_policy.rb

```diff
@@ -0,0 +1,51 @@
+########################################################################
+# AwsSqsQueuePolicy is the +aws_sqs_queue_policy+ terrform resource,
+#
+# {https://www.terraform.io/docs/providers/aws/r/sqs_queue_policy.html Terraform Docs}
+########################################################################
+class GeoEngineer::Resources::AwsSqsQueuePolicy < GeoEngineer::Resource
+  validate -> { validate_required_attributes([:queue_url, :policy]) }
+
+  after :initialize, -> {
```

Comment:
Typically this is a one-liner if there isn't a really long value for `_terraform_id`, i.e

```ruby
after :initialize, -> { _terraform_id -> { queue_url } }
```